downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1432 @@
+    "factor": 1.0,
+    "owner": "admin"
+};
+
+type ExclusiveCongifuration record {|
+    // The record type descriptor of  `ExclusiveCongifuration` overrides the rest 
+    // descriptor from `Congifuration`, making the rest descriptor of `ExclusiveCongifuration` 
+    // be of type `boolean`.
+    *Congifuration;
+    boolean active;
+    boolean...;
+|};
+
+ExclusiveCongifuration exclusiveConfig = {
+    id: 1003,
+    active: false,
+    // Additional fields of type `boolean` can be specified since the rest descriptor is overridden.
+    "allow": true
+};
+```
+
+The rest descriptor type of the `InclusiveCongifuration` is `anydata` and that of the `ExclusiveCongifuration` is `boolean`. The including records override the rest descriptor from the included record.
+
+##### Improved Listener Declaration  
+
+The listener declaration is improved to accept listener values that are a subtype of both the compiler built-in type `listener` and `error` type. This allows the following listener declaration when the `init` method of `Listener` may return an error.
+
+```ballerina
+listener lsn = new Listener();
+```
+
+Before this change, `listener` authors had to use `panic` in the `init` method instead of returning an error if the `listener` object failed to initialize.
+
+If the initialization of the listener object returns an error, module initialization will fail.
+
+```ballerina
+listener lsn = new Listener();
+
+public class Listener {
+
+   public isolated function 'start() returns error? {
+   }
+
+   public isolated function gracefulStop() returns error? {
+   }
+
+   public isolated function immediateStop() returns error? {
+   }
+
+   public isolated function detach(service object {} s) returns error? {
+   }
+
+   public isolated function attach(service object {} s, string[]|string? name = ()) returns error? {
+   }
+
+   public function init() returns error? {
+      ...
+      // object init failed due to some reason
+      if (object_init_failed) {
+         return error("Listener initialization failed");
+      }
+   }
+}
+```
+
+**Previous Syntax:**
+```ballerina
+listener lsn = new Listener();
+
+public class Listener {
+   ...
+   ...
+   // listener life cycle methods
+
+   public function init() {
+      ...
+      // object init failed due to some reason
+      if (object_init_failed) {
+         panic error("Listener initialization failed");
+      }
+   }
+}
+```
+
+##### Referring Lang Library Modules Without Using Quoted Identifiers
+
+Lang library module prefixes can now be used without the initial quote. For example, both the approaches below are now supported.
+
+```ballerina
+int a = int:sum(1, 2);
+```
+
+```ballerina
+int a = ‘int:sum(1, 2);
+```
+
+The way of overriding a langlib module is also relaxed now. For example, both the approaches below are now supported.
+
+```ballerina
+import somePackage.someModule as string;
+```
+
+and
+
+```ballerina
+import somePackage.someModule as `string;
+```
```

Comment:
Do we have to include this in the release note? @hasithaa @rdulmina

Reply:
Yes, IMO it's kind of a highlight

Reply:
This is probably not something commonly done though, right? And I'm not sure if we should be encouraging it.

Reply:
@hasithaa - WDYT?

Reply:
+1, to remove it. Since this is not encouraging.

Reply:
Removed.